### ISSUE TITLE
Accessible names are missing WPF DataGridHyperlinkColumn cells

### ISSUE THREAD

**Issue by guybark** [microsoft/dotnet#1203]
Create a WPF DataGrid with a DataGridHyperlinkColumn, based on the snippets up at [DataGridHyperlinkColumn Class](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.datagridhyperlinkcolumn?view=netframework-4.8). Point the Accessibility Insights for Windows tool at the link cell, and notice that it reports that the UI Automation elements representing both the cell and the contained link have no UIA Names, (as shown in the image below). In line with this, if I tab to either the cell or the contained link, the Narrator screen reader doesn’t announce the text shown visually in the cell.

I know the data grid supports actions to bind the AutomationProperties.Name property on a cell showing an image, but in this case both the link cell and the contained link need to have the AutomationProperties.Name property Set.

![AIWinLink](https://user-images.githubusercontent.com/14315611/81741660-bfd7aa80-9453-11ea-9c5b-b68021836b42.jpg)
